pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -224,7 +224,10 @@ def test_conn_exists(context):
     Exception:
        click.Exception if no current server.
     """
-    if not context.conn:
+    # TODO/KS: Is this necessary since we should be protecting with classes
+    #          or does the pywbem_server access provide same protection??
```

Comment:
Accessing ContextObj.pywbem_server already checks for a server, so this additional check is not needed.

Reply:
Removed in the Part 2 PR

Reply:
ok